stix_shifter_modules/crowdstrike_logscale/README.md

```diff
@@ -68,228 +77,265 @@ translate crowdstrike_logscale query "{}" "[process:name IN ('cmd.exe','calc.exe
+                            "kill_parent": "false",
+                            "kill_process": "false",
+                            "kill_subprocess": "false",
+                            "operation_blocked": "false",
+                            "policy_disabled": "false",
+                            "process_blocked": "true",
+                            "quarantine_file": "false",
+                            "quarantine_machine": "false",
+                            "registry_operation_blocked": "false",
+                            "rooting": "false",
+                            "sensor_only": "false",
+                            "suspend_parent": "false",
+                            "suspend_process": "false"
+                        },
+                        "rule_instance_id": "3",
+                        "rule_instance_version": "3",
+                        "scenario": "suspicious_activity",
+                        "severity": "50",
+                        "sha256": "1010101010101010101010100101100101010101010101010101010101010100",
+                        "tactic": "Custom Intelligence",
+                        "tactic_id": "CSTA0005",
+                        "technique": "Indicator of Attack",
+                        "technique_id": "CST0004",
+                        "template_instance_id": "3",
+                        "timestamp": "2024-01-23T12:33:07Z",
+                        "triggering_process_graph_id": "pid:7adb:123",
+                        "user_id": "S-1-5-18",
+                        "user_name": "user1"
+                    }
+                ],
+                "behaviors_processed": [
+                    "pid:7adb:123:41002"
+                ],
+                "cid": "123",
+                "created_timestamp": "2024-01-23T12:33:15.170758259Z",
+                "date_updated": "2024-03-27T14:22:28.062194Z",
+                "detection_id": "ldt:xyz:123",
+                "device": {
+                    "agent_load_flags": "1",
+                    "agent_local_time": "2024-01-23T12:32:59.287Z",
+                    "agent_version": "7.05.17706.0",
+                    "bios_manufacturer": "Xen",
+                    "bios_version": "4.11.amazon",
+                    "cid": "123",
+                    "config_id_base": "65994763",
+                    "config_id_build": "17706",
+                    "config_id_platform": "3",
+                    "device_id": "7adb123",
+                    "external_ip": "1.1.1.1",
+                    "first_seen": "2023-05-16T05:10:55Z",
+                    "groups": [
+                        "97350feebe4541e8a615c0d3f18acdf3",
+                        "bb1e1190b46348e69e10785030e8b23d"
+                    ],
+                    "hostname": "host",
+                    "instance_id": "i-123",
+                    "last_login_timestamp": "2024-01-04T06:12:09Z",
+                    "last_login_user": "Administrator",
```

Comment:
This should probably be obscured (if it's not already).

Reply:
Updated the value